pkg/shared/utils.go

```diff
@@ -264,3 +264,15 @@ func ExtractRepositoryInfoFromURL(Url string, VCSPlugName string) (string, strin
 
 	return "", "", "", "", "", fmt.Errorf("invalid URL: %s", Url)
 }
+
+func IsCI() bool {
```

Comment:
@shikari-ac , check this

Reply:
Anyway you've done it. I am ok

Reply:
I decided, that it make sense to make it unified among multiple plugins.